### PR TITLE
Feature/word not in list

### PR DESCRIPTION
**Pull request by hothanhtuan1209**
In this pull request, i refactor word_not_in_list.py file in practice-2, update function main and add module word_not_in_list to main.py
#50

### CODE REVIEW

File: python/practices/practice-3/src/path2/word_not_in_list.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+prints all the words from the ebook that do not appear in the word list.
+"""
+
+
+import os
+from constants.constants import EBOOK, WORD
+from helpers.file_reader import read_file
+
+
+def extract_words_from_content(content):
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/word_not_in_list.py]: I think `extract_words` is good name.

**Comment by hothanhtuan1209** [comment]
image

File: python/practices/practice-3/src/path2/word_not_in_list.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    for word in content.split():
+        cleaned_word = word.strip()
+
+        if cleaned_word:
+            word_list.append(cleaned_word)
+
+    return word_list
+
+
+def find_missing_words(book_content, words_content):
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/word_not_in_list.py:31]: Please update the docstring.

File: python/practices/practice-3/src/path2/word_not_in_list.py (hunk 1 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/word_not_in_list.py]: Done

File: python/practices/practice-3/src/path2/word_not_in_list.py (hunk 2 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/word_not_in_list.py:31]: Done

File: python/practices/practice-3/src/path2/word_not_in_list.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+        list: A list of words extracted from the content.
+    """
+
+    word_list = []
+
+    for word in content.split():
+        cleaned_word = word.strip()
+
+        if cleaned_word:
+            word_list.append(cleaned_word)
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/word_not_in_list.py]: Can you refactor this code?
Ex: work_list = [word for work in content.split() if word.strip()]

File: python/practices/practice-3/src/path2/word_not_in_list.py [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+              words_content.
+    """
+
+    test_words = extract_words(book_content)
+    words_list = extract_words(words_content)
+    missing_words = []
+
+    for word in test_words:
+        if word not in words_list:
+            missing_words.append(word)
```
Comment (cuongdoan-agilityio) [inline python/practices/practice-3/src/path2/word_not_in_list.py]: Similarly, refactor for this code.

File: python/practices/practice-3/src/path2/word_not_in_list.py (hunk 3 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/word_not_in_list.py]: Done

File: python/practices/practice-3/src/path2/word_not_in_list.py (hunk 4 above)
Comment (hothanhtuan1209) [reply python/practices/practice-3/src/path2/word_not_in_list.py]: Done

**Pull request merged**